BeagleBone_black/src/fake_imu_input_acc.txt

```diff
@@ -1,3 +1,14 @@
-000 1.0 1.0 1.0 10.0 10.0 10.0 
```

Comment:
Can you move this file out of `src/` folder? Do not mix code with data. You can create a new `BeagleBone_blakc/data/in/` folder, for example.